### ISSUE TITLE
Feature request: show latest version in admin panel

### ISSUE THREAD

**Issue by lonix1** [wekan/wekan#1920]
The version info was added in the admin panel because of [this issue](https://github.com/wekan/wekan/issues/963).

But what is missing is a "latest version" field, which shows the latest stable version from github. Not everyone is using snapd, so it's good to know when wekan can be upgraded.

Ideas
- To keep it simple it could be a button that does an ajax request to the "latest releases" page and shows the current and latest version - i.e. on demand
- More complex is to run this check once in a while, and show a popup somewhere to say that a new version is available (but only do this "call-home feature" if user gives permission in admin panel)
